src/main/kotlin/turku/forge/babyproject/config/WebSocketConfig.kt

```diff
@@ -5,21 +5,26 @@ import org.springframework.messaging.simp.config.MessageBrokerRegistry
 import org.springframework.web.socket.config.annotation.EnableWebSocketMessageBroker
 import org.springframework.web.socket.config.annotation.StompEndpointRegistry
 import org.springframework.web.socket.config.annotation.WebSocketMessageBrokerConfigurer
+import turku.forge.babyproject.api.API_PATH
 
-const val STOMP_ENDPOINT = "/connect"
-const val APPLICATION_DESTINATIONS = "/message"
+
+const val STOMP_ENDPOINT = "$API_PATH/connect"
+const val SIMPLE_MESSAGE_BROKER_ENDPOINT = "$API_PATH/channel"
 
 /**
  * This is a configuration where we extend and override the base `WebSocketMessageBrokerConfigurer`
  */
 @Configuration
 @EnableWebSocketMessageBroker
 class WebSocketConfig : WebSocketMessageBrokerConfigurer {
-    override fun registerStompEndpoints(registry: StompEndpointRegistry) {
-       registry.addEndpoint( STOMP_ENDPOINT )
+    override fun configureMessageBroker(config: MessageBrokerRegistry) {
+        config.enableSimpleBroker(SIMPLE_MESSAGE_BROKER_ENDPOINT)
     }
 
-    override fun configureMessageBroker(registry: MessageBrokerRegistry) {
-        registry.setApplicationDestinationPrefixes( APPLICATION_DESTINATIONS )
+    override fun registerStompEndpoints(registry: StompEndpointRegistry) {
+        registry.addEndpoint(STOMP_ENDPOINT)
+                // @TODO: get allowed origins from config file
```

Comment:
I think we should avoid having `TODO`s in our code; it's extra noise and they're often left there indefinitely. They make for perfect issues instead!

Reply:
Ahh but this is a love letter to my self so I don't forget what I need to do before this implementasion is done